dace/libraries/stencil/intel_fpga.py

```diff
@@ -198,7 +198,52 @@ def expansion(node, parent_state, parent_sdfg):
             connector = field_to_edge[field_name].dst_conn
             data_name_outer = connector
             data_name_inner = field_name + "_in"
-            desc_outer = parent_sdfg.arrays[data_name].clone()
+            parent_desc = parent_sdfg.arrays[data_name]
+            if isinstance(parent_desc, dt.View):
+                if isinstance(parent_desc, dt.ArrayView):
+                    desc_outer = dt.Array(parent_desc.dtype,
```

Comment:
doesn't arrayview have an "as_array" method? Same applies for the rest of this file.
This will be problematic if we decide to change the fields in data containers. I am trying to avoid such issues.